### PR TITLE
Don't rename main file when cached entry is a directory

### PR DESCRIPTION
**Pull request by michaelva**

### CODE REVIEW

File: nuxeo-core/nuxeo-core-convert/src/main/java/org/nuxeo/ecm/core/convert/service/ConversionServiceImpl.java [hunk 1]
```diff
@@ -323,7 +323,7 @@ public BlobHolder convert(String converterName, BlobHolder blobHolder, Map<Strin
             if (config.isCacheEnabled()) {
                 ConversionCacheHolder.addToCache(cacheKey, result);
             }
-        } else {
+        } else if (result.getBlobs().size()==1){
```
[the remark this answers is not in the document]
Comment (michaelva) [reply nuxeo-core/nuxeo-core-convert/src/main/java/org/nuxeo/ecm/core/convert/service/ConversionServiceImpl.java]: The comment was already just below :)

**Comment by michaelva** [comment]
This is not a new behavior. However the code wasn't restrictive enough such that the filename was emptied more broadly than the intended scope. Non regression is covered by the existing unit tests.

**Comment by michaelva** [comment]
Updated the jira issue

**Comment by kevinleturc** [comment]
T&P : https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-kleturc/462/

**Comment by efge** [comment]
👍

**Pull request merged**